resources/assets/uad_lists.json

```diff
@@ -19730,11 +19730,11 @@
   {
     "id": "com.lenovo.ue.device",
     "list": "Oem",
-    "description": "User Experience Program\nAnalytics stuff. Displays an annoying notification after every reboot prompting you to join this user experience program.",
+    "description": "User Experience Program\nAnalytics stuff. Displays an annoying notification after every reboot prompting you to join this user experience program.\nNOTE: removing this package may cause a factory reset, see:\nhttps://github.com/0x192/universal-android-debloater/issues/713.",
```

Comment:
```suggestion
    "description": "User Experience Program\nAnalytics stuff. Displays an annoying notification after every reboot prompting you to join Lenovo's user experience program.\nNOTE: removing this package may cause a hardware reset after unlocking the device, see:\nhttps://github.com/0x192/universal-android-debloater/issues/713.",
```

Reply:
Aahh okay, so you mean a hard reboot. When I hear people saying reset, I always think of a factory reset. Maybe we should also add what you said here:

> If USB debugging is still active you can safely reinstall the package and the device works again, but if USB debugging is disabled you might fail to do so - what leaves you no option but to perform an actual factory reset.

If you can edit your suggestion, I'll commit your suggestion. 👍🏻